addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2069,6 +2069,13 @@ const Wysiwyg = Widget.extend({
             ? finalOptions.autohideToolbar
             : !finalOptions.snippets;
 
+        // TODO: remove this on master, disable option in the views instead.
+        // Disable image conversion to attachment for mail and mass_mailing,
+        // as they handle the conversion themselves.
+        const resModel = (options.recordInfo && options.recordInfo.res_model) || "";
+        if (['mail.', 'mailing.'].some(module =>  resModel.startsWith(module))) {
+            finalOptions.dropImageAsAttachment = false;
+        }
```

Comment:
I had an idea this morning. I think those models are marked as requiring inline style conversion, otherwise they may not display properly in all email clients. Same for the user signature. Therefore, I'm not sure that we need to generate an access token anymore because the models that require it, that is things that are going to be sent by email, should have this option disabled as they have the attachments generated through convert inline.